plugins/modules/omevv/omevv_firmware.py

```diff
@@ -0,0 +1,689 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.10.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: omevv_firmware
+short_description: Update the firmware of a specific host in the cluster
+version_added: "9.10.0"
+description: This module allows you to update the firmware of a specific host in the cluster.
+extends_documentation_fragment:
+  - dellemc.openmanage.omevv_auth_options
+options:
+  check_vSAN_health:
+    description:
+      - Whether to check vSAN health whilst update.
+      - C(true) checks the vSAN health whilst update.
+      - C(false) does not check the vSAN health whilst update.
+    type: bool
+  date_time:
+    description:
+      - Date and time when the job must run. This is applicable when I(run_now) is C(false).
+      - The format is YYYY-MM-DDThh:mm:ss<offset>.
+    type: str
+  delete_job_queue:
+    description:
+      - Whether to delete the job queue in iDRAC while performing firmware update.
+      - C(true) deletes the job queue in iDRAC while performing firmware update.
+      - C(false) does not delete the job queue in iDRAC while performing firmware update.
+    type: bool
+  drs_check:
+    description:
+      - Allows to check if cluster's DRS is enabled or not.
+      - C(true) checks if cluster's DRS is enabled.
+      - C(false) does not check if cluster's DRS is enabled.
+    type: bool
+    default: false
+  enter_maintenance_mode_options:
+    description:
+      - VM migration policy during management mode.
+      - C(FULL_DATA_MIGRATION) for full data migration.
+      - C(ENSURE_ACCESSIBILITY) for ensuring accessibility.
+      - C(NO_DATA_MIGRATION) does not migrate any data.
+    type: str
+    choices: [FULL_DATA_MIGRATION, ENSURE_ACCESSIBILITY, NO_DATA_MIGRATION]
+  enter_maintenance_mode_timeout:
+    description:
+      - Time out value during maintenance mode in minutes.
+    type: int
+    default: 60
+  evacuate_VMs:
+    description:
+      - Allows to move the VM to other host when host is powered off.
```

Comment:
•	Allows you to move the VM to another host when the current host is powered off.
•	C(true) moves the VM to another host when the current host is powered off.
•	C(false) does not move the VM to another host when the current host is powered off.

Reply:
Incorporated